smarts/core/agent_manager.py

```diff
@@ -223,25 +223,27 @@ def observe(
                 else:
                     logging.log(
                         logging.DEBUG,
-                        f"Agent `{agent_id}` has raised done with {observations[agent_id].events}",
+                        # pytype: disable=attribute-error
+                        f"Agent `{agent_id}` has raised done with {observations[agent_id].events}", 
+                        # pytype: enable=attribute-error
                     )
 
-                rewards[agent_id] = vehicle.trip_meter_sensor(increment=True)
-                scores[agent_id] = vehicle.trip_meter_sensor()
+                rewards[agent_id] = float(vehicle.trip_meter_sensor(increment=True))
+                scores[agent_id] = float(vehicle.trip_meter_sensor())
 
         if sim.should_reset:
             dones = {agent_id: True for agent_id in self.agent_ids}
             dones["__sim__"] = True
 
-        return observations, rewards, scores, dones
+        return observations, rewards, scores, dones 
```

Comment:
extra whitespace

Reply:
I believe such whitespaces will be automatically corrected by the `isort, Black, and prettier` step in `ci-auto-commit-linux.yml`. The `ci-auto-commit-linux.yml` file runs and commits automatically upon merging a pull request into the `develop` branch.